src/components/optimizerTab/Sidebar.jsx

```diff
@@ -303,6 +326,14 @@ function MobileSidebarContent() {
               size={[8, 5]}
               percent={Math.floor(Number(permutationsSearched) / Number(permutations) * 100)}
             />
+            <Flex>
+              <Typography>Time remaining:</Typography>
+              <Divider style={{ margin: 'auto 10px', flexGrow: 1, width: 'unset', minWidth: 'unset' }} dashed />
+              <Typography>{optimizationInProgress
+                ? Utils.msToReadable((permutations / (permutationsSearched ? permutationsSearched : 1) * (Date.now() - startTime)) - (Date.now() - startTime))
```

Comment:
This formula felt weird to me, my reaction was it should be something like (ms per permutation) * (permutations left) unless there's something I'm missing in this

`msDiff / permutationsSearched * (permutations - permutationsSearched)`